src/test/java/com/wikia/webdriver/testcases/mercurytests/ImageReviewTests.java

```diff
@@ -0,0 +1,123 @@
+package com.wikia.webdriver.testcases.mercurytests;
+
+import com.wikia.webdriver.common.core.Assertion;
+import com.wikia.webdriver.common.core.annotations.Execute;
+import com.wikia.webdriver.common.core.helpers.User;
+import com.wikia.webdriver.common.templates.NewTestTemplate;
+import com.wikia.webdriver.elements.mercury.pages.ImageReview;
+
+import org.testng.annotations.Test;
+
+@Test(groups = "ImageReview")
+public class ImageReviewTests extends NewTestTemplate {
+
+  private final int STAFF_NAVBAR_BUTTONS_NUMBER = 2;
+  private final int REVIEWER_NAVBAR_BUTTONS_NUMBER = 1;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Rename this field "REVIEWER_NAVBAR_BUTTONS_NUMBER" to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS00116)
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Make this final field static too. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS1170)